Executive.cpp

```diff
@@ -0,0 +1,86 @@
+/*
+
+Humberto Dominguez
+Professor Scott Roueche
+CSE 687- Object Oriented Design
+Syracuse University
+Phase 1 - Executive cpp
+4/20/2023
+
+This class was created in part of creating the MapReduce code for Phase 1 for
+CSE 687. The Executive class's primary purpose is to gather all data related to directories.
+
+Since this is the cpp portion of the class, this will define how and where the the member and methods will be
+used.
+*/
+
+//libraries that will be used in this class
+#include "Executive.h"
+
+//default constructor 
+Executive::Executive() {}
+
+//default destructor 
+Executive::~Executive() {}
+
+//mathod that will obtain the directories and test to verify they exist
+void Executive::obtainingDirectories() {
+	
+    bool checker = 0; //This is a bool variable to verify that the input is correct
+    do {
+        cout << "Please enter the input directory where you have stored the files: "; //asking for user input
+        getline(cin, inputDir); // recording input
+        checker = checkInput(inputDir); 
+    } while (checker == 0);
+
+    //since it is a do while loop, we don't have to manually change it to 0, since it will have to run through at leats once
+    //checking intermediate file directory
+    do {

```

Comment:
we should check to see that all 3 directories aren't in the same location.

Reply:
If they are in the same directory what are we going to do? make them switch, or give them a warning?